pkg/nsx/services/common/types.go

```diff
@@ -15,83 +15,85 @@ import (
-	DefaultVMSubnetSet                 string = "vm-default"
-	SystemVPCNetworkConfigurationName  string = "system"
-	TagScopeSubnetCRUID                string = "nsx-op/subnet_uid"
-	TagScopeSubnetCRName               string = "nsx-op/subnet_name"
-	TagScopeSubnetSetCRName            string = "nsx-op/subnetset_name"
-	TagScopeSubnetSetCRUID             string = "nsx-op/subnetset_uid"
-	TagValueGroupScope                 string = "scope"
-	TagValueGroupSource                string = "source"
-	TagValueGroupDestination           string = "destination"
-	TagValueShareCreatedForInfra       string = "infra"
-	TagValueShareCreatedForProject     string = "project"
-	TagValueShareNotCreated            string = "notShared"
-	TagValueGroupAvi                   string = "avi"
-	TagValueSLB                        string = "SLB"
-	AnnotationVPCNetworkConfig         string = "nsx.vmware.com/vpc_network_config"
-	AnnotationSharedVPCNamespace       string = "nsx.vmware.com/shared_vpc_namespace"
-	AnnotationDefaultNetworkConfig     string = "nsx.vmware.com/default"
-	AnnotationAttachmentRef            string = "nsx.vmware.com/attachment_ref"
-	TagScopePodName                    string = "nsx-op/pod_name"
-	TagScopePodUID                     string = "nsx-op/pod_uid"
-	ValueMajorVersion                  string = "1"
-	ValueMinorVersion                  string = "0"
-	ValuePatchVersion                  string = "0"
-	ConnectorUnderline                 string = "_"
+	HashLength                           int    = 8
+	MaxTagScopeLength                    int    = 128
+	MaxTagValueLength                    int    = 256
+	MaxIdLength                          int    = 255
+	MaxNameLength                        int    = 255
+	MaxSubnetNameLength                  int    = 80
+	PriorityNetworkPolicyAllowRule       int    = 2010
+	PriorityNetworkPolicyIsolationRule   int    = 2090
+	TagScopeNCPCluster                   string = "ncp/cluster"
+	TagScopeNCPProjectUID                string = "ncp/project_uid"
+	TagScopeNCPVIFProjectUID             string = "ncp/vif_project_uid"
+	TagScopeNCPPod                       string = "ncp/pod"
+	TagScopeNCPVNETInterface             string = "ncp/vnet_interface"
+	TagScopeCreatedFor                   string = "nsx-op/created_for"
+	TagScopeVersion                      string = "nsx-op/version"
+	TagScopeCluster                      string = "nsx-op/cluster"
+	TagScopeNamespace                    string = "nsx-op/namespace"
+	TagScopeNamespaceUID                 string = "nsx-op/namespace_uid"
+	TagScopeSecurityPolicyCRName         string = "nsx-op/security_policy_cr_name"
+	TagScopeSecurityPolicyCRUID          string = "nsx-op/security_policy_cr_uid"
+	TagScopeSecurityPolicyName           string = "nsx-op/security_policy_name"
+	TagScopeSecurityPolicyUID            string = "nsx-op/security_policy_uid"
+	TagScopeSecurityPolicyNamespacedName string = "nsx-op/security_policy_namespaced_name"
```

Comment:
The TagScopeSecurityPolicyNamespacedName includes both 'namesapce's name/obj name'.
Then it has the redundant info. 
Is it possible to remove TagScopeSecurityPolicyCRName?
or use TagScopeSecurityPolicyNamespace +  TagScopeSecurityPolicyCRName  to filter the securitypolicy?

Reply:
Thanks for advice, as you suggested, we user namespace + CRname to filter which  securitypolicy should be deleted.